packages/react-dom/src/__tests__/ReactServerRenderingHydration-test.js

```diff
@@ -301,10 +299,8 @@ describe('ReactDOMServerHydration', () => {
         );
       });
     }).toErrorDev(
-      'Warning: Prop `style` did not match. Server: ' +
-        '{"text-decoration":"none","color":"black","height":"10px"}' +
-        ' Client: ' +
-        '{"textDecoration":"none","color":"black","height":"10px"}', // note that this is no difference
+      "A tree hydrated but some attributes of the server rendered HTML didn't match the client properties.",
```

Comment:
maybe drop "rendered"? easy to read it as a verb and then the sentence structure breaks